Lib9c/BlockPolicySource.cs

```diff
@@ -69,14 +69,15 @@ public IBlockPolicy<NCAction> GetPolicy(int minimumDifficulty)
             BlockChain<NCAction> blockChain
         )
         {
-            if (transaction.Actions.Count == 1 &&
-                transaction.Actions.First().InnerAction is ActivateAccount)
-            {
-                return true;
-            }
-
             try
             {
+                if (transaction.Actions.Count == 1 &&
+                    transaction.Actions.First().InnerAction is ActivateAccount aa)
+                {
+                    return blockChain.GetState(aa.PendingAddress) is Dictionary rawPending &&
```

Comment:
Can this throw `IncompleteBlockStatesException`? If so, it is okay to return true on the case where it actually fails to verify but the state is still not calculated?

Reply:
> Can this throw IncompleteBlockStatesException?

Yes.

> If so, it is okay to return true on the case where it actually fails to verify but the state is still not calculated?

It can be matter. `IncompleteBlockStatesException` will be thrown during IBD. but there is no method to workaround at this time. (See also: https://github.com/planetarium/lib9c/pull/89)

I guess, the way we can solve this problem is by dropping legacy block state store and features for state downloading and forcing state calculation even if IBD too.

Reply:
> I guess, the way we can solve this problem is by dropping legacy block state store and features for state downloading and forcing state calculation even if IBD too.

Great. Can we add FIXME on here or make an issue referencing your suggestion?